vcd/resource_vcd_vapp.go

```diff
@@ -172,94 +168,84 @@ func resourceVcdVAppCreate(d *schema.ResourceData, meta interface{}) error {
 			}
 
-			err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-				networks := []map[string]interface{}{map[string]interface{}{
-					"ip":         d.Get("ip").(string),
-					"is_primary": true,
-					"orgnetwork": d.Get("network_name").(string),
-				}}
-				task, err := vapp.ChangeNetworkConfig(networks, d.Get("ip").(string))
-				if err != nil {
-					return resource.RetryableError(fmt.Errorf("error with Networking change: %#v", err))
-				}
-				return resource.RetryableError(task.WaitTaskCompletion())
-			})
+			networks := []map[string]interface{}{map[string]interface{}{
+				"ip":         d.Get("ip").(string),
+				"is_primary": true,
+				"orgnetwork": d.Get("network_name").(string),
+			}}
+			task, err = vapp.ChangeNetworkConfig(networks, d.Get("ip").(string))
+			if err != nil {
+				return fmt.Errorf("error with Networking change: %#v", err)
+			}
+			err = task.WaitTaskCompletion()
 			if err != nil {
 				return fmt.Errorf("error changing network: %#v", err)
 			}
 
 			if ovf, ok := d.GetOk("ovf"); ok {
-				err := retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-					task, err := vapp.SetOvf(convertToStringMap(ovf.(map[string]interface{})))
-
-					if err != nil {
-						return resource.RetryableError(fmt.Errorf("error set ovf: %#v", err))
-					}
-					return resource.RetryableError(task.WaitTaskCompletion())
-				})
+				task, err := vapp.SetOvf(convertToStringMap(ovf.(map[string]interface{})))
+
+				if err != nil {
+					return fmt.Errorf("error set ovf: %#v", err)
+				}
+				err = task.WaitTaskCompletion()
 				if err != nil {
 					return fmt.Errorf("error completing tasks: %#v", err)
 				}
 			}
 
 			initscript, ok := d.GetOk("initscript")
 			if ok {
-				err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-					log.Printf("running customisation script")
-					task, err := vapp.RunCustomizationScript(d.Get("name").(string), initscript.(string))
-					if err != nil {
-						return resource.RetryableError(fmt.Errorf("error with setting init script: %#v", err))
-					}
-					return resource.RetryableError(task.WaitTaskCompletion())
-				})
+				log.Printf("running customisation script")
+				task, err := vapp.RunCustomizationScript(d.Get("name").(string), initscript.(string))
+				if err != nil {
+					return fmt.Errorf("error with setting init script: %#v", err)
+				}
+				err = task.WaitTaskCompletion()
 				if err != nil {
 					return fmt.Errorf(errorCompletingTask, err)
 				}
 			}
 
 			if d.Get("power_on").(bool) {
-				err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-					task, err := vapp.PowerOn()
-					if err != nil {
-						return resource.RetryableError(fmt.Errorf("error powerOn machine: %#v", err))
-					}
-					return resource.RetryableError(task.WaitTaskCompletion())
-				})
+
+				task, err := vapp.PowerOn()
+				if err != nil {
+					return fmt.Errorf("error powerOn machine: %#v", err)
```

Comment:
"error powerOn machine" -> ""error power on the machine"

Reply:
Fixed